classes/ActionScheduler_AdminView.php

```diff
@@ -110,6 +110,89 @@ protected function get_list_table() {
 		return $this->list_table;
 	}
 
+	/**
+	 * Action: admin_notices
+	 *
+	 * Maybe check past-due actions, and print notice.
+	 *
+	 * @uses $this->check_pastdue_actions()
+	 */
+	public function maybe_check_pastdue_actions() {
+
+		# Filter to prevent checking actions (ex: inappropriate user).
+		if ( !apply_filters( 'action_scheduler_check_pastdue_actions', true ) )
```

Comment:
We follow the WP coding standard of `if ( ! apply....`. There's a second one of these below.

Reply:
@rrennick My apologies; fixed in b3e80b5.